packages/design-system/src/components/panel.tsx

```diff
@@ -4,7 +4,8 @@ export const panelStyles = css({
   overflow: "auto",
   backgroundColor: "$slate4",
   borderRadius: "$1",
-  boxShadow: "0px 2px 7px rgba(0, 0, 0, 0.1), 0px 5px 17px rgba(0, 0, 0, 0.15)",
```

Comment:
I remember having updated this shadow from figma, just a few days ago, lets 
1. discuss how this is wrong
2. normalize asking questions if you see we are back and forth on something

Reply:
We seem to be looking at different figma, lets figure this out:

## case 1: 

https://www.figma.com/file/EfzZ2w94KpmUviZJSBgxCd/Designer-UI?node-id=1602%3A41906

```
box-shadow: 0px 2px 7px rgba(0, 0, 0, 0.1), 0px 5px 17px rgba(0, 0, 0, 0.15);
```

<img width="368" alt="Screenshot 2022-11-01 at 10 37 39" src="https://user-images.githubusercontent.com/52824/199204694-d20be31a-5102-4cd0-a805-6d863a46c753.png">


## case 2

https://www.figma.com/file/EfzZ2w94KpmUviZJSBgxCd/Designer-UI?node-id=2327%3A47295

```
box-shadow: 0px 2px 7px rgba(0, 0, 0, 0.1), 0px 5px 17px rgba(0, 0, 0, 0.15);
```

<img width="294" alt="Screenshot 2022-11-01 at 10 40 11" src="https://user-images.githubusercontent.com/52824/199205097-30599a09-f8d5-4fec-aa7a-234a19cc745e.png">

Reply:
@natlusyht is @taylornowotny helpful? I think he confirmed what I was saying

Reply:
ohhh now I understand

Reply:
btw background color is gray4 instead of slate4, it was already wrong, not a regression, but since we looked please take it along